### PR TITLE
13 14 15 implement job offer mv and employer router

### PR DESCRIPTION
**Pull request by macibol775**
please note that there are some lacking features so we had to use some plugs

### CODE REVIEW

File: public/js/modifyOffer.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    const formData = new FormData(form);
+    console.log(formData);
+    // request.addEventListener('load', (event) => {
+    //     const response = JSON.parse(event.target.responseText);
+    //     console.log(response);
+    // });
+    
+    request.open('put', '/employer/add_offer');
+    request.send(formData);
+});
```
Comment (WojciechPtas) [inline public/js/modifyOffer.js:16]: add endline on the end of the file

File: routes/mainRoutes.js [hunk 2]
```diff
@@ -32,9 +32,17 @@ const profilePicUploadMiddleware = (req, res, next) => {
...
+router.post('/profile/docs_upload', docUploadMiddleware, profileController.docs_upload_post)
+router.get('/profile', profileController.profile_get);
+router.post('/profile', profilePicUploadMiddleware, profileController.profile_post);
+router.delete('/profile/file_delete', upload.none(), profileController.file_delete);
+=======
 router.post('/profile/docs_upload', [docUploadMiddleware, authMiddleware.require_login], profileController.docs_upload_post);
 router.get('/profile', authMiddleware.require_login, profileController.profile_get);
 router.post('/profile', [profilePicUploadMiddleware, authMiddleware.require_login], profileController.profile_post);
 router.delete('/profile/file_delete', authMiddleware.require_login, profileController.file_delete);
+>>>>>>> main
```
Comment (tomazaw) [inline routes/mainRoutes.js:46]: Left after conflict resolve

**Review by tomazaw — CHANGES_REQUESTED** [review]

File: routes/mainRoutes.js [hunk 3]
```diff
@@ -32,9 +32,17 @@ const profilePicUploadMiddleware = (req, res, next) => {
     });
 }
 
+<<<<<<< HEAD
```
Comment (MasterCoookie) [inline routes/mainRoutes.js:35]: Remove merge conflict pozostałości

**Review by MasterCoookie — CHANGES_REQUESTED** [review]
Remove github śmeci and unnecessary scraper

File: public/js/modifyOffer.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    const request = new XMLHttpRequest();
+    const formData = new FormData(form);
+    console.log(formData);
+    // request.addEventListener('load', (event) => {
+    //     const response = JSON.parse(event.target.responseText);
+    //     console.log(response);
+    // });
+    
+    request.open('put', '/employer/add_offer');
+    request.send(formData);
```
[the remark this answers is not in the document]
Comment (MasterCoookie) [reply public/js/modifyOffer.js:14]: Good, bu in future make it visible for the user

**Review by MasterCoookie — APPROVED** [review]
In future, make server responses visible for user

**Review by tomazaw — APPROVED** [review]

**Pull request merged**